CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 1.1.0 (February 23, 2023)
+
+SECURITY:
+
+* Update Envoy to 1.25.1 within the Dockerfile. [[GH-71](https://github.com/hashicorp/consul-dataplane/pull/71)]
+* Upgrade golang/x/net to 0.7.0
+This resolves vulnerability [CVE-2022-41723](https://github.com/golang/go/issues/57855) in `x/net` [[GH-81](https://github.com/hashicorp/consul-dataplane/pull/81)]
+* Upgrade to use Go 1.20.1. 
+This resolves vulnerabilities [CVE-2022-41724](https://go.dev/issue/58001) in `crypto/tls` and [CVE-2022-41723](https://go.dev/issue/57855) in `net/http`. [[GH-78](https://github.com/hashicorp/consul-dataplane/pull/78)]
+
+FEATURES:
+
+* support Envoy admin [access logs](https://developer.hashicorp.com/consul/docs/connect/observability/access-logs). [[GH-65](https://github.com/hashicorp/consul-dataplane/pull/65)]
```

Comment:
Is it expected that the docs link here doesn't work (yet at least)?

Reply:
Yes, this will be available when Consul 1.15 moves into "production" on CRT.